cmakeast/ast.py

```diff
@@ -137,7 +138,7 @@ def __repr__(self):
 # do a lot of type checking to make sure that you don't compare
 # enums of different types. Since we could be analyzing
 # quite a lot of code, performance is more important than safety here.
-class WordType(object):  # suppress(R0903,too-few-public-methods)
+class WordType(IntEnum):  # suppress(R0903,too-few-public-methods)
```

Comment:
Have a look at the comment above - did you performance test this code?

Reply:
No, I didn't, but I expect it to be a bit worse, since `IntEnum` members are members of a real class in python, not a dumb numbers. Though I think convenience when debugging (`IntEnum` shows names when `repr`ed) is more important here than performance.